docs/services/kafka.md

```diff
@@ -104,3 +104,147 @@ Pillowtop process configurations as described in the [CommCare docs](https://com
     ```
 
 See https://kafka.apache.org/documentation.html#basic_ops_cluster_expansion for more details.
+
+### Replication
+
+#### [kafka-reassign-tool](https://github.com/dimas/kafka-reassign-tool)
+
+A helper script for Kafka to make it easier changing replicas for existing topics
```

Comment:
could give a very short overview here and then point to the github docs for more detail rather than copying the docs here.

Reply:
Updated